tests/test_all.py

```diff
@@ -6,6 +6,7 @@
 
 from construct import *
 from construct.lib import *
+from construct.core import UnionError
```

Comment:
Ah that might be it, go ahead and add it. Looks like another small bug found and fixed. Wow. Piles up.